MNE/disp3D/3DObjects/bem/bemsurfacetreeitem.cpp

```diff
@@ -97,6 +97,8 @@ BemSurfaceTreeItem::BemSurfaceTreeItem(int iType, const QString& text)
 
 BemSurfaceTreeItem::~BemSurfaceTreeItem()
 {
+    setVisible(false);
+    delete m_pRenderable3DEntity;
```

Comment:
good so far. Is m_pRenderable3DEntity a QObejct. If this is the case consider using a QPointer isntead of raw pointer. No explicit deletion would be reuqired anyomre. - how about m_pParenEntity - Is this handles by the parent?